### PR TITLE
Login page - dodatočné úpravy

### PR DESCRIPTION
**Pull request by batsysk**
Added some small changes in login.blade and auth.blade

### CODE REVIEW

File: src/resources/views/layout/auth.blade.php [hunk 1]
```diff
@@ -31,13 +31,10 @@ class="light-style customizer-hide"
...
     <div class="authentication-wrapper authentication-basic container-p-y">
         <div class="authentication-inner">
-            @yield('register')
+            @yield('content')
         </div>
     </div>
 </div>
-@yield('content')
-
-@yield('js')
```
Comment (LukasHajda) [inline src/resources/views/layout/auth.blade.php:38]: - Treba tam doplnit login.min.js file
- a taktiez yield('js') keb bolo potrebne doplnit zvlast JS kod z view

**Review by LukasHajda — COMMENTED** [review]
Treba doplnit kod

**Comment by batsysk** [comment]
added changes after review

**Review by LukasHajda — APPROVED** [review]

**Review by RMankovecky — APPROVED** [review]
good job

**Pull request merged**